client/src/components/admin/CategoryManagement.tsx

```diff
@@ -0,0 +1,486 @@
+import React, { useState, useEffect } from 'react';
+
+interface Category {
```

Comment:
I want to the right word for this table in stead of generic name as `Category`

```suggestion
interface Vessel {
```